JULIERAJ/KIDS-FIRST-Front
# Feature/86/popup choose kid color

## One Line Description
Module window to choose kid color

## Requirements
- Build module as per [Figma](https://www.figma.com/design/qUBlZE5TIgpqShKTVcSvRo/Kids-First-WITT(Desktop)?node-id=12526-51263&t=6Ak6TUUF92a4Hp4m-4) requirements 

## Notes
Thanks to Ekaterina for building the module. All that was left to be done was integrating the module into the form Kid's form.

## Test Steps
1. Go to http://localhost:3000/dashboard/kids  
2. Click on the color edit button
3. Choose different colors and confirm if the Circles on the left side of the module both change accordingly (no profile picture integration as of yet)
4. Close the module - note that the Kid's color has not changed
5. Click again on the Edit button
6. Choose a different color and press the Save button. The module should close automatically
7. Note that the Kid's color has changed.

## UI/UX Outcome
[2024-09-15, 2_09_23 p.m. - Screen - Untitled video.webm](https://github.com/user-attachments/assets/afe77a44-250e-4d81-83f7-8a3813f41d16)
